contracts/test/DepositStub.sol

```diff
@@ -85,4 +85,8 @@ contract DepositStub is IDeposit {
     function withdrawableAmount() external view override returns (uint256) {
         return address(this).balance;
     }
+
+    function notifyFraud() external {
```

Comment:
This is not a part of a real Deposit contract public API? If so, we should add a note about it, just like we did with `setAuctionValue`.

Reply:
Also, linter is complaining about the function order, can we fix it?

Reply:
It's under:
```
    ///
    /// Not in tBTC deposit interface, added just for tests.
    ///
```

Regarding the order, I'll change.

Reply:
No, this comment is now just for that one function, because `withdrawableAmount` is a part of the public interface and it is located below. I had to chang the approach because linter was annoying in PRs :/

Reply:
Ok, done: https://github.com/keep-network/coverage-pools/pull/109/commits/363e308394a507077a3c4aeffaba22c7cfe6798b